gpep517/scheme.py

```diff
@@ -0,0 +1,154 @@
+# SPDX-FileCopyrightText:
+# SPDX-License-Identifier: MIT
+
+# Most of this file is copied from https://github.com/pypa/installer/
+# with minor tweaks where classes don't have hooks to control their
+# internals.
+
+from __future__ import annotations
+
+import contextlib, io, os, shlex, zipfile
```

Comment:
Split imports.

Reply:
Bit disappointing tbh... all it does is make files longer and more annoying to scroll down to the actual content, but so be it.